acc_test/import_resource_test.go

```diff
@@ -131,39 +131,39 @@ func TestAccKafkaCluster_importBasic(t *testing.T) {
 	})
 }
 
-func TestAccEncryptionKey_importBasic(t *testing.T) {
-	testAccEBSKeyProvider := instaclustr.Provider()
-	testAccEBSKeyProviders := map[string]terraform.ResourceProvider{
-		"instaclustr": testAccEBSKeyProvider,
-	}
-	validConfig, _ := ioutil.ReadFile("data/valid_encryption_key.tf")
-	username := os.Getenv("IC_USERNAME")
-	apiKey := os.Getenv("IC_API_KEY")
-	hostname := getOptionalEnv("IC_API_URL", instaclustr.DefaultApiHostname)
-	providerAccountName := os.Getenv("IC_PROV_ACC_NAME")
-	kmsArn := os.Getenv("KMS_ARN")
-	oriConfig := fmt.Sprintf(string(validConfig), username, apiKey, hostname, kmsArn, providerAccountName)
-	// Not running this test parallelly since we only have 1 test encryption key
-	resource.Test(t, resource.TestCase{
-		Providers:    testAccEBSKeyProviders,
-		CheckDestroy: testCheckAccEBSResourceDeleted("valid", hostname, username, apiKey),
-		Steps: []resource.TestStep{
-			{
-				Config: oriConfig,
-				Check: resource.ComposeTestCheckFunc(
-					testCheckAccEBSResourceValid("valid"),
-					testCheckAccEBSResourceCreated("valid", hostname, username, apiKey),
-				),
-			},
-			{
-				Config:            oriConfig,
-				ResourceName:      "instaclustr_encryption_key.valid",
-				ImportState:       true,
-				ImportStateVerify: true,
-			},
-		},
-	})
-}
+//func TestAccEncryptionKey_importBasic(t *testing.T) {
```

Comment:
Maybe @arjunrajnetapp can confirm, but is the process to just comment out the tests? shouldnt we be deleting them?  commented code always gives me shivers

Reply:
We'll be getting rid of all of these tests come v2, but in my opinion I'd prefer to comment these out than outright delete them in the off chance that we want to turn them back on between now and releasing v2.